Installer/setup.iss

```diff
@@ -39,8 +39,8 @@ Name: "jre"; Description: "Java Runtime Environment version 8."; Types: full
 
 [Run]
 ; install if frameworks needed - Check: CheckForFramework;
-Filename: {tmp}\dotNetFx40_setup.exe; Parameters: "/q:a /c:""install /l /q"""; Check: CheckForFramework; StatusMsg: Microsoft Framework 4.0 is be�ng installed. Please wait...
-Filename: {tmp}\vcredist_x86.exe; Parameters: "/q:a /c:""install /l /q"""; Check: VCSetupNeeded; StatusMsg: Microsoft Visual C++ 2010 is be�ng installed. Please wait...
+Filename: {tmp}\dotNetFx40_setup.exe; Parameters: "/q:a /c:""install /l /q"""; Check: CheckForFramework; StatusMsg: Microsoft Framework 4.0 is be�ng installed. Please wait...
```

Comment:
what changed in these lines? `be�ng installed` is encoding error...

Reply:
hmm, probably because I edited it on Linux with another encoding?
Could you try to fix it on your machine?